src/libs/Navigation/linkingConfig.js

```diff
@@ -25,11 +25,7 @@ export default {
                     [SCREENS.REPORT]: ROUTES.REPORT_WITH_ID,
                 },
             },
-            [NAVIGATORS.FULL_SCREEN_NAVIGATOR]: {
```

Comment:
I think we should remove the `FULL_SCREEN_NAVIGATOR` from `src/Navigators.js` 
https://github.com/Expensify/App/blob/f4ab2459091061c02046df0ff542457700067d6d/src/NAVIGATORS.js#L8